vscode-trace-extension/src/trace-explorer/opened-traces/trace-explorer-opened-traces-webview-provider.ts

```diff
@@ -132,15 +132,18 @@ export class TraceExplorerOpenedTracesViewProvider implements vscode.WebviewView
                     case VSCODE_MESSAGES.OPEN_TRACE:
                         vscode.commands.executeCommand('openedTraces.openTraceFolder');
                         return;
-                    case VSCODE_MESSAGES.EXPERIMENT_SELECTED: {
-                        let experiment: Experiment | undefined;
-                        if (message.data && message.data.wrapper) {
-                            experiment = convertSignalExperiment(JSONBig.parse(message.data.wrapper));
-                        } else {
-                            experiment = undefined;
+                    case VSCODE_MESSAGES.EXPERIMENT_SELECTED:
+                        // Need to block scope the variable 'experiment'...
+                        if (true) {
+                            let experiment: Experiment | undefined;
+                            if (message.data && message.data.wrapper) {
+                                experiment = convertSignalExperiment(JSONBig.parse(message.data.wrapper));
+                            } else {
+                                experiment = undefined;
+                            }
+                            signalManager().fireExperimentSelectedSignal(experiment);
                         }
-                        signalManager().fireExperimentSelectedSignal(experiment);
-                    }
+                        return;
```

Comment:
Experiment is defined in code above, and I was getting a `tsc` error because 'experiment' was already defined.  So I had to block-scope this to appease the compiler.  I did not change any logic or code otherwise...

Reply:
No need of an` if(true)` statement, just wrap the switch cases with braces so that any variable declared within that block is only visible within the block.

Reply:
I just removed this code from the PR.